src/views/collection-view/server.ts

```diff
@@ -22,16 +22,22 @@ export interface CollectionPagePath {
   }
 }
 
-export type CollectionPageProduct = TutorialPageProduct
-
+/**
+ * Given a ProductData object (imported from src/data JSON files) and a
+ * Collection slug, , fetches and returns the page props for
+ * `/{productSlug}/tutorials/{collectionSlug}` pages.
+ *
+ * Returns the given ProductData object unmodified as the `product` page prop,
+ * which is needed for other areas of the app to function.
+ */
```

Comment:
Will let this comment do the initial explaining, and absolutely happy to answer any questions. 💖